src/js/components/StarRating/StarRating.js

```diff
@@ -0,0 +1,40 @@
+import React, { useContext, useState } from 'react';
+import { ThemeContext } from 'styled-components';
+import { Star, StarOutline } from 'grommet-icons';
+import { RadioButtonGroup } from '../RadioButtonGroup';
+
+const StarRating = ({ ...rest }) => {
+  const theme = useContext(ThemeContext);
+  const [value, setValue] = useState();
+
+  const options = [];
+  for (let i = 0; i < 5; i += 1) {
```

Comment:
I see here that we're indexing from 0 (which I know is standard for an array), but I'm wondering if it would make more sense to start at `1`? Because a value of 0 almost feels like the user didn't select anything. Soft opinion.

Reply:
Im good either way Ive seen both ways when I was looking around

Reply:
changed it 👍